hack/convert-output-to-tsv.py

```diff
@@ -31,134 +31,149 @@ def printd(x):
+                                printd(f"......adding {col} to split_column_set ({col_counter}) ...{len(column_set)} columns left")
+                        # remove the split_column_set from column_set
+                        column_set = [x for x in column_set 
+                                     if x not in split_column_set]
+                        split_column_list = sorted(list(split_column_set))
+                        # add PK to each split as first column                
+                        split_column_list.insert(0, entity_name)
+                        print(f"......Split #{chunk} now contains {len(split_column_list)} columns")
+                        printd(f"cols in split: {len(split_column_list)}")
+                        printd(f"cols left to split: {len(column_set)}")
+                        # split rows
+                        split_row_dict_list = list()
+                        # iterate through every row looking for every column for this split
+                        for row in row_list:
+                            split_row_dict = dict()
+                            for col in split_column_list:
+                                # if a whitelisted col exists in this row
+                                if (col in dict(row).keys()):
+                                    # add the col to the split list
+                                    # clean up whitespace
+                                    try:
+                                        split_row_dict[col] = (dict(row).get(col)).replace('\r\n',' ').strip()
+                                    except AttributeError:
+                                        split_row_dict[col] = (dict(row).get(col))
+                                else:
+                                    pass
+                            # add the row to the list of rows for this
+                            split_row_dict_list.append(split_row_dict)
+                        # output to tsv
+                        dw = csv.DictWriter(output_file, split_column_list, delimiter='\t')
+                        dw.writeheader()
+                        dw.writerows(split_row_dict_list)
+                    print(f"......{table_name} Split #{chunk} was successfully written to {output_location}")
+            else:
+                # this thread is executed for any tables with 512 cols or less
+                print(f"...No need to split files for {table_name}")
+                output_location = output_dir + '/' + table_name + '.tsv'
+                print(f"...Writing {table_name} to {output_location}")
+                with open(output_location, 'w') as output_file:
+                    dw = csv.DictWriter(output_file, sorted_column_set, delimiter='\t')
+                    dw.writeheader()
+                    dw.writerows(row_list)
+                print(f"...{table_name} was successfully written to {output_location}")
+    except OSError as e:
```

Comment:
:-1: big try-catch blocks like this are a code smell and can make issues harder to debug. let's try to identify lines where we expect errors and want to skip over them (do we, since all we're doing is printing the error?) and put try-catch blocks around them instead.

Reply:
I've walked back these changes minus just adding the environment table logic.